client/src/app/site/base/base-import.service/base-backend-import.service.ts

```diff
@@ -176,7 +176,7 @@ export abstract class BaseBackendImportService implements BackendImportService {
             header: true,
             skipEmptyLines: `greedy`,
             quoteChar: this.textSeparator,
-            transform: value => (!value ? undefined : value)
+            transform: value => (!value ? undefined : value.trim())
```

Comment:
`value.trim()` could be false while `value` is true. Therefore you should also use `value.trim()` for the check.